apps/prairielearn/elements/pl-matching/pl-matching.css

```diff
@@ -47,3 +47,35 @@
   background-color: rgba(0, 0, 0, 0.03);
   margin-right: 0.5rem;
 }
+
+.pl-matching-container.right {
```

Comment:
I'm a bit worried about using a class with such a common name, as it may conflict with other classes in use in other elements or by instructor code. I'd suggest having a class that is clearly specific to the element, such as `pl-matching-container-right` instead of `right`.